.github/workflows/image.yaml

```diff
@@ -0,0 +1,15 @@
+name: Images
+on:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  push:
```

Comment:
It might not be necessary to rebuild the artifacts on push. I would consider a separate workflow for push later down the line.